opentelemetry-sdk/src/opentelemetry/sdk/_metrics/export/__init__.py

```diff
@@ -176,3 +178,49 @@ def _shutdown():
         self._daemon_thread.join()
         self._exporter.shutdown()
         return True
+
+
+class PullMetricExporter(MetricExporter):
+    def __init__(self) -> None:
+        self._collect: Callable[
+            ["PullMetricExporter"], None,
+        ] = None
+
+    @final
+    def _set_collect_callback(
+        self,
+        func: Callable[
+            ["PullMetricExporter"], None,
+        ],
+    ) -> None:
+        """This function is internal to the SDK. It should not be called or overriden by users"""
+        self._collect = func
+
```

Comment:
Should there be `export` here?

Reply:
No, it remains an abstract method because the PullMetricExporter isn't an implementation of an exporter itself.